src/router/storeSync/storeSync.config.js

```diff
@@ -119,6 +120,22 @@ const storeSyncConfig = [
         Boolean,
         false
     ),
+    new SimpleUrlParamConfig(
+        '3d',
+        'setShowIn3d',
+        'setShowIn3d',
+        (store) => store.state.ui.showIn3d,
+        false,
+        Boolean,
+        false
+    ),
```

Comment:
this is now a duplicate, you might have added that in your latest rebase
please remove it

Reply:
thanks for finding this.